cmd/cli/serve/serve.go

```diff
@@ -286,13 +281,31 @@ func serve(cmd *cobra.Command) error {
 		return fmt.Errorf("error starting node: %w", err)
 	}
 
+	startWebUI, err := config.Get[bool](types.NodeWebUIEnabled)
+	if err != nil {
+		return err
+	}
+
 	// Start up Dashboard
 	if startWebUI {
+		listenPort, err := config.Get[int](types.NodeWebUIPort)
+		if err != nil {
+			return err
+		}
+
+		// If we get the default value for an int, use 80 instead.
+		if listenPort == 0 {
+			listenPort = 80
+		}
```

Comment:
Some day it would be nice to be able to set 0 to mean "let the kernel pick a free port"